apidiff.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Could we name this file to something like `generate-api-diff.sh` to make it slightly more descriptive? I think it would also make more sense to be in the `.utility` folder, which would involve moving directories a little bit in the script.